README.md

```diff
@@ -40,6 +40,9 @@ class YourComponent extends React.Component {
             onLogout={() => this.setState({ accessToken: null, filepath: null })}
             onFilePick={filepath => this.setState({ filepath })}
             onError={error => console.error(error.message)}
+            loggedInMsg={loggedInMsg}
+            loggedOutMsg={loggedOutMsg}
+            fileSelectedMsg={fileSelectedMsg}
```

Comment:
Do you mind adding the `fileChangeWarningMsg` to the README?

Reply:
Nevermind.. I'm creating a new PR to make it easier to contribute in the future. I'll add it there.